### ISSUE TITLE
Provider install error for version 0.180.0

### ISSUE THREAD

**Issue by mohammadasim** [aliyun/terraform-provider-alicloud#5279]
Hi there,

The new version of terraform Alibaba cloud provider has broken our environment. We use terraform cloud to run our scripts.
### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.
`Terraform 1.1.9`

### Affected Resource(s)
The problem appears when I run the command `terraform init`

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://gist.github.com/mohammadasim/de8842948651bc71cad56161467d4af6

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
What should have happened?
I expected the provider to be installed without error.

### Actual Behavior
What actually happened?
The provider didn't install even after running the `terraform init` command several times.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `Write code that would provision resources in Alibaba cloud`
2. `set your provider version to v0.180.0`
3. `terraform init`

### Important Factoids
NA

### References
NA

**Comment by supertom25** [comment]
encounter same issue here. need to enforce the source to aliyun/alicloud as workaround.......but not nice we have over 100+ repository and managed by different teams. difficult to communicate the changes so sudden. could you please resolve the broken link

**Comment by mohammadasim** [comment]
> aliyun/alicloud

We are using TFC. I will appreciate if you could explain your workarount. In our case version 0.180.0 is required by the state. As things stand we totally blocked by this issue.
```
Providers required by configuration:
.
├── provider[terraform.io/builtin/terraform]
├── provider[registry.terraform.io/aliyun/alicloud] 1.160.0
├── provider[registry.terraform.io/hashicorp/kubernetes] 1.13.4
├── provider[registry.terraform.io/hashicorp/aws] 4.18.0
└── module.redis
    ├── provider[registry.terraform.io/aliyun/alicloud] 1.160.0
    └── module.accounts
        └── provider[registry.terraform.io/aliyun/alicloud] 1.160.0

Providers required by state:

    provider[terraform.io/builtin/terraform]

    provider[registry.terraform.io/aliyun/alicloud]

    provider[registry.terraform.io/hashicorp/alicloud]

    provider[registry.terraform.io/hashicorp/aws]
```

**Comment by mohammadasim** [comment]
> > aliyun/alicloud
> 
> We are using TFC. I will appreciate if you could explain your workarount. In our case version 0.180.0 is required by the state. As things stand we totally blocked by this issue.
> 
> ```
> Providers required by configuration:
> .
> ├── provider[terraform.io/builtin/terraform]
> ├── provider[registry.terraform.io/aliyun/alicloud] 1.160.0
> ├── provider[registry.terraform.io/hashicorp/kubernetes] 1.13.4
> ├── provider[registry.terraform.io/hashicorp/aws] 4.18.0
> └── module.redis
>     ├── provider[registry.terraform.io/aliyun/alicloud] 1.160.0
>     └── module.accounts
>         └── provider[registry.terraform.io/aliyun/alicloud] 1.160.0
> 
> Providers required by state:
> 
>     provider[terraform.io/builtin/terraform]
> 
>     provider[registry.terraform.io/aliyun/alicloud]
> 
>     provider[registry.terraform.io/hashicorp/alicloud]
> 
>     provider[registry.terraform.io/hashicorp/aws]
> ```

I managed to fix the issue using the following command:
```
terraform state replace-provider registry.terraform.io/hashicorp/alicloud registry.terraform.io/aliyun/alicloud
```

**Comment by supertom25** [comment]
@mohammadasim yes same here to use state replace-provider to switch the provider in state.
next you need to define the provider source in terraform block.

**Comment by supertom25** [comment]
@mohammadasim yes same here to use state replace-provider to switch the provider in state.
next you need to define the provider source in terraform block.
